Mapping/Loader/LoaderChain.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Algolia\AlgoliaSearchBundle\Mapping\Loader;
+
+use Doctrine\ORM\EntityManager;
+
+final class LoaderChain implements LoaderInterface
+{
+    private $loaders;
+
+    public function getMetaData($entity, EntityManager $em)
+    {
+        foreach ($this->loaders as $loader) {
+            if (null !== $metadata = $loader->getMetaData($entity, $em)) {
```

Comment:
To be honest, I did this patch more than 2 months ago and I am not planning to re-work this anytime soon as it's working great in production.
Feel free to patch my work.